lib/openscap_parser/group.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+module OpenscapParser
+  class Group < XmlNode
+    include OpenscapParser::Util
+
+    def id
+      @id ||= parsed_xml['id']
+    end
+
+    def title
+      @title ||= parsed_xml.at_css('title') &&
+        parsed_xml.at_css('title').text
+    end
+
+    def description
+      @description ||= newline_to_whitespace(
+        parsed_xml.at_css('description') &&
+          parsed_xml.at_css('description').text || ''
+      )
+    end
+
+    def rationale
+      @rationale ||= newline_to_whitespace(
+        parsed_xml.at_css('rationale') &&
+          parsed_xml.at_css('rationale').text || ''
+      )
+    end
+
+    def requires
+      @requires ||= parsed_xml.xpath('./requires') &&
+        parsed_xml.xpath('./requires/@idref').flat_map do |r|
+          r.to_s&.split
+        end
+    end
+
+    def conflicts
+      @conflicts ||= parsed_xml.xpath('./conflicts') &&
+        parsed_xml.xpath('./conflicts/@idref').flat_map do |c|
+          c.to_s&.split
+        end
+    end
```

Comment:
@skateman I updated these methods because I realized that the way that it was set up before wouldn't work correctly if a Rule or Group had more than one `conflicts` or `requires` element under it, which is possible.
So if there was a Rule or Group with more than one `conflicts` or `requires` element under it, the `idref`s were being merged together into one long string and they weren't being saved correctly during the import
I fixed the code and the tests in compliance-backend so this should be covered now!

Reply:
:slightly_smiling_face: seems like the idea of having 2 PRs on both sides helped, at least a bit

Reply:
It's definitely has helped! It would've been a lot harder to test and go back and forth making updates without 2 PRs open 🙂